doc/CustomEventConnector..md

```diff
@@ -0,0 +1,77 @@
+# reshuffle
+
+[Code](https://github.com/reshufflehq/reshuffle) |
+[npm](https://www.npmjs.com/package/reshuffle) |
+[Code sample](https://github.com/reshufflehq/reshuffle/examples/customEvent)
+
+
+### Reshuffle Custom Event Connector
+
+This package contains a [Reshuffle](https://github.com/reshufflehq/reshuffle)
+connector that can fire a customized event. Developer can fire it from a script.
+
+The following example exposes an endpoint to return the data of a Custom event after it was fired.
+
+```js
+const { CustomEventConnector, Reshuffle } = require('reshuffle')
+
+const app = new Reshuffle()
+
+const connector = new CustomEventConnector(app)
+
+connector.on(
+    { channel:'channel-one', payload: { name: 'Jack', age: 25 } },
+    (event) => {
+      console.log('Custom Event One: ', event.options)
+    }
+  )
+  
+  connector.on(
+    { channel:'channel-two', payload: 'A String to display' },
+    (event) => {
+      console.log('Custom Event Two: ', event.options)
+    },
+  )
+
+app.start()
+```
+
+### Table of Contents
+
+#### Connector Events
+
+[Fire Custom Events](#listen)
```

Comment:
I think 'Fire Custom Events' can be confusing as an event name. I was not sure whether this is an event or an action. Looking at the rest of the readme - I think the event and action were mixed into one...